scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -49,17 +51,21 @@ def __init__(self,
                 test. This is the main case associated with the test.
             separate_builds (bool): Whether separate builds are needed for the
                 two cases. If False, case2 uses the case1 executable.
+            separate_builds (bool): Whether separate runs are needed for the
+                two cases. If False, case2 uses the case1 run area. Can be useful for restart tests
             run_two_suffix (str, optional): Suffix appended to the case name for
                 the second run. Defaults to 'test'. This can be anything other
                 than 'base'.
             run_one_description (str, optional): Description printed to log file
                 when starting the first run. Defaults to ''.
             run_two_description (str, optional): Description printed to log file
                 when starting the second run. Defaults to ''.
+            multisubmit (bool): Do first and second runs as different submissions
```

Comment:
Yes, it is confusing. I believe the value of RESUBMIT in case2 is irrelevant. The resubmission logic is contained within the original case.